scripts/common/rook.sh

```diff
@@ -173,3 +173,238 @@ function maybe_cleanup_rook() {
 function rook_osd_phase_ready() {
     [ "$(kubectl -n rook-ceph get cephcluster rook-ceph --template '{{.status.phase}}')" = 'Ready' ]
 }
+
+function current_rook_version() {
+    kubectl -n rook-ceph get deploy rook-ceph-operator -oyaml 2>/dev/null \
```

Comment:
i question whether or not we want to hide the error here since we check that rook is installed prior to calling this function

Reply:
I could go either way
I pulled this directly from the rook code 😆